products/sle/main.pm

```diff
@@ -458,6 +458,9 @@ sub load_inst_tests() {
             loadtest 'installation/partitioning_full_lvm';
         }
         if (get_var("FILESYSTEM")) {
+            if (get_var('PARTITIONING_WARNINGS')) {
+                loadtest 'installation/partitioning_warnings';
+            }
```

Comment:
no it's not about me not wanting to test opneSUSE, it's about time and priorities. When I worked on something that YaST changed you asked me to work on something more important.
This feature is explicitly  [SLES-12-SP3 Ready](https://fate.suse.com/320416), of course I can spent maybe another day checking all opneSUSE versions
My development machine is idle, I have nothing to do